partner_contact_personal_information_page/README.rst

```diff
@@ -42,6 +65,7 @@ Contributors
 * Jairo Llopis <j.llopis@grupoesoc.es>
 * Matjaž Mozetič <m.mozetic@matmoz.si>
 * Rudolf Schnapka <schnapkar@golive-saar.de>
+* Richard deMeester <richard@willowit.com.au>
 
 Maintainer
```

Comment:
Sorry if I was not accurate enough. Here is the expected standard text for the maintainer section:
Maintainer
Odoo Community Association

This module is maintained by the OCA.

OCA, or the Odoo Community Association, is a nonprofit organization whose mission is to support the collaborative development of Odoo features and promote its widespread use.

To contribute to this module, please visit http://odoo-community.org.

Please check the standard OCA REadme file here: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

Let me know if it is clearer